00-repos-centos8.ks

```diff
@@ -0,0 +1,6 @@
+url --mirrorlist=http://mirrorlist.centos.org/?release=8&arch=$basearch&repo=baseos
+repo --name="AppStream" --mirrorlist=http://mirrorlist.centos.org/?release=8&arch=$basearch&repo=appstream
+repo --name="epel8" --mirrorlist=https://mirrors.fedoraproject.org/mirrorlist?repo=epel-8&arch=x86_64
```

Comment:
is this really needed? we don't support foreman installations on el8 with epel enabled.

Reply:
Oh likely not, that was because of facter 2.x. Building without to confirm.

Reply:
Yes this works fine, EPEL was not necessary.